dev-util/gnome-builder/gnome-builder-46.3.ebuild

```diff
@@ -0,0 +1,274 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..12} )
+DISABLE_AUTOFORMATTING=1
+FORCE_PRINT_ELOG=1
+
+inherit gnome.org gnome2-utils llvm meson optfeature python-single-r1 readme.gentoo-r1 virtualx xdg
+
+DESCRIPTION="An IDE for writing GNOME-based software"
+HOMEPAGE="https://wiki.gnome.org/Apps/Builder https://gitlab.gnome.org/GNOME/gnome-builder"
```

Comment:
Please use the newer web:
https://apps.gnome.org/Builder/

Reply:
Done in https://github.com/gentoo/gentoo/pull/34771/commits/2e79d01ce9014f788953f80376df87ba8186e743